nilearn/decoding/space_net.py

```diff
@@ -1134,6 +1131,27 @@ def _binarize_y(self, y):
         self.n_classes_ = len(self.classes_)
         return y
 
+    def score(self, X, y, sample_weight=None):
+        """Returns the mean accuracy on the given test data and labels.
+
+        Parameters
+        ----------
+        X : list of Niimg-like objects
+            See http://nilearn.github.io/building_blocks/
```

Comment:
I was wondering whether we could not use a HTML redirect:
http://stackoverflow.com/questions/5411538/redirect-from-an-html-page

This way we could have a shorter URL that fits on one line and additional level of indirection in case this part of the documentation moves around.

Anyone who want to give it a try, be my guest!